src/Type/ObjectType.ts

```diff
@@ -8,7 +9,10 @@ export class ObjectProperty {
     ) {
     }
 
-    public getName(): string {
+    public getName(stripQuotes: boolean = false): string {
+        if (stripQuotes) {
```

Comment:
Why don't we always strip quotes?

Reply:
Not explicit reason, just wanted to scope my change to the problem at hand as I'm not 100% familiar with the library and didn't want to cause any side-effects.

Reply:
I'm fairly confident that it should be okay to always strip quotes.